using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-event-reference.md

```diff
@@ -76,7 +74,7 @@ The Campaign Classic extension also reads the following key from the configurati
 
 The following events are dispatched by the Campaign Classic extension:
 
-### Campaign Response Content
+### Campaign response
```

Comment:
Campaign response content

Reply:
This is not fixed still, use `Campaign response content`